apps/dashboard/src/components/cmd-k-search/index.tsx

```diff
@@ -226,8 +226,8 @@ export const CmdKSearch: React.FC = () => {
       {/* modal below here */}
 
       <DialogContent
-        className="z-[10000001] p-0 gap-0"
-        dialogOverlayClassName="z-[10000000] backdrop-blur-sm"
+        className="p-0 gap-0"
+        dialogOverlayClassName="backdrop-blur-sm"
```

Comment:
last time I asked @jakubkrehel about "hey should we use backdrop-blur` he mentioned we likely shouldn't - can u check with him on that here? (I know we already have it but might as well)

Reply:
Removed